doc/debian/jitsi-meet/jitsi-meet.example

```diff
@@ -40,8 +40,11 @@ server {
     error_page 404 /static/404.html;
 
     gzip on;
-    gzip_types text/plain text/css application/javascript application/json;
+    gzip_types text/plain text/css application/javascript application/json image/jpeg image/png image/x-icon application/octet-stream audio/mpeg audio/mpeg;
```

Comment:
Why would you want to zip mp3? Does it really help?

Reply:
Hmmm good point, it could help slightly but not worth the CPU cycles - removed it